### PR TITLE
[WebUI] 图形化配置页面

### PR DESCRIPTION
**Pull request by Gaojianli**
close https://github.com/PBH-BTN/PeerBanHelper/issues/527

## Release Notes

- **New Features**
  - Introduced a settings page with a tabbed interface for configuration and user profile settings.
  - Added multiple new components for managing settings, including buttons, IP database, logging, performance, and proxy settings.
  - Implemented localization for settings in both English and Chinese, enhancing user experience.
  - New asynchronous functions for managing user profiles and configurations.
  - Added a structured form for managing configuration settings, including language preferences and privacy options.
  - Updated project version to 2.4.0 and added a new script for linting.
  - Introduced components for managing auto-range bans, client name blacklists, and peer ID blacklists.

- **Bug Fixes**
  - Updated localization strings for better clarity and consistency.

- **Documentation**
  - Enhanced user understanding of settings with new localized string constants.

- **Chores**
  - Updated development dependencies to ensure improved performance and stability.

### CODE REVIEW

**Comment by Gaojianli** [comment]
@coderabbitai review this

**Comment by Ghost-chu** [comment]
@coderabbitai review

File: webui/src/views/settings/components/config/components/webui.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      :rules="[
+        { type: 'url', required: true },
+        { match: /[^\/]$/, message: t('page.settings.tab.config.server.prefix.error') }
+      ]"
+    >
+      <a-input v-model="model.prefix" style="width: 200px" />
+    </a-form-item>
+    <a-form-item label="Token" field="server.token" required>
+      <a-input-password v-model="model.token" style="width: 350px" />
+    </a-form-item>
```
[the remark this answers is not in the document]
Comment (Gaojianli) [reply webui/src/views/settings/components/config/components/webui.vue:53]: 瞎几把胡扯，根本就没有这个东西

File: webui/src/views/settings/components/profile/components/formArray.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    </a-space>
+  </a-form-item>
+</template>
+<script setup lang="ts">
+import type { PaginationProps } from '@arco-design/web-vue'
+import type { VirtualListProps } from '@arco-design/web-vue/es/_components/virtual-list-v2/interface'
+import { computed } from 'vue'
+
+type Value = string
+const model = defineModel<Value[]>({ required: true })
```
[the remark this answers is not in the document]
Comment (Gaojianli) [reply webui/src/views/settings/components/profile/components/formArray.vue:52]: defineModel  is a marco, it don't need to be imported

**Comment by Gaojianli** [comment]
@coderabbitai review again

**Comment by Gaojianli** [comment]
@coderabbitai most review comment resolved, please review again

**Pull request merged**